public/locales/cs/translation.json

```diff
@@ -266,7 +266,8 @@
 		"Invited": "Pozván",
 		"Resend invitation": "Přeposlat pozvánku",
 		"Something went wrong, failed to resend invitation": "Něco je špatně, pozvánka nebyla odeslána",
-		"Invitation sent successfully": "Pozvánka úspěšně odeslána"
+		"Invitation sent successfully": "Pozvánka úspěšně odeslána",
+		"Editing is disabled for this credential": "Editace je pro tohoto uživatele zakázána"
```

Comment:
@aringocode Please ask @byewokko for correct alert message and translation

Reply:
"Tohoto uživatele není možné editovat."
also in english i'd prefer something along the lines of "These credentials are not editable."

i'd avoid "disabled" or "zakázáno" because it suggests that it could somehow be "enabled", which is not true. it is simply not editable and the user cannot do anything about it.

Reply:
or possibly
"Credentials do not support editing."
"Editace tohoto uživatele není podporována."